package.json

```diff
@@ -96,8 +98,9 @@
     "node-sass": "^4.12.0",
     "nyc": "^14.1.1",
     "raw-loader": "^0.5.1",
-    "react": "^0.14 || ^15.5.4 || ^16.1.1",
-    "react-dom": "^0.14 || ^15.5.4 || ^16.1.1",
+    "react": "^16.8.6",
```

Comment:
these make it a breaking change; you may want to revert the react and react-dom lines if that's not the goal